docs/content/getting-started/production-environment.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Running OpenFGA in Production
+description: Running OpenFGA in a Production Environment
+sidebar_position: 8
+slug: /getting-started/running-in-production
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+} from "@components/Docs";
+
+# Running OpenFGA in Production
+
+<DocumentationNotice />
+
+## Preparation
+
+If you have tried OpenFGA locally, decided it works as expected, and now want to deploy it to production, we recommend the following preparation:
+
+- Update the configuration as follows:
+  - [Configure authentication](./setup-openfga.mdx#configuring-authentication) on the server.
+  - Enable TLS on the server.
+  - Enable production logging by changing the `log.format` in the [config.yaml](#configuring-the-server) configuration file to `json`.
+  - [Disable Playground](./setup-openfga.mdx#playground).
+- Add a rate limiter interceptor (for example, [in the server configuration](https://pkg.go.dev/github.com/openfga/openfga/server#Config)) that protects your OpenFGA server against bursts in traffic.
+- Add a logger interceptor (for example, [in the server configuration](https://pkg.go.dev/github.com/openfga/openfga/server#Config)) that sends logs to your logging platform of choice.
```

Comment:
Same comment as above. This isn't really possible when running OpenFGA as a standalone server.

We don't yet have a pluggable middleware mechanism for this kind of stuff.